<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>detectorMain</class>
 <widget class="QMainWindow" name="detectorMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>detectorMain</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="DeviceCountLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>This will show the device count</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GetDeviceCountButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Device Count</string>
    </property>
   </widget>
   <widget class="QPushButton" name="beginRecordingButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Begin Recording</string>
    </property>
   </widget>
   <widget class="QLabel" name="noteLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Note Played</string>
    </property>
   </widget>
   <widget class="QLabel" name="barsLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>551</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Bars #</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopRecordingButton">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>310</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Recording</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
